docs/resources/agent_to_agent.md

```diff
@@ -8,57 +8,57 @@ description: |-
 
 # thousandeyes_agent_to_agent (Resource)
 
+This resource allows you to create and configure an agent-to-agent test. This test type evaluates the performance of the underlying network between two physical sites. For more information about agent-to-agent tests, see [Agent-to-Agent Tests](https://docs.thousandeyes.com/product-documentation/internet-and-wan-monitoring/tests#agent-to-agent-test).
 
-
-
+## Example Usage
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `agents` (Block Set, Min: 1) Agents to use (see [below for nested schema](#nestedblock--agents))
```

Comment:
Everything under ` <!-- schema generated by tfplugindocs -->` will be overwritten by `go generate`. These descriptions will need to be changed in [schemas.go](https://github.com/thousandeyes/terraform-provider-thousandeyes/blob/496382e168f660e21f5a640f2632658af043506e/thousandeyes/schemas.go).